### DISCUSSION TITLE
Difference between Lenze_ST17H66 and AirTag

### DISCUSSION THREAD

**Discussion by paolostefanucci** [Chapoly1305/FindMy#13] [Q&A] [answered]
What is the difference between  Lenze_ST17H66 Tag and real Apple AirTag?
Beyond the advertisement keys  Does the AirTag send other data? Like battery status, airtag firmware version etc.?
How I can add more information data on Lenze_ST17H66 Tag advertisement? Like battery status, firmware version, etc.?
Are there some bits free on the advertisement data?
Apple could in some way block the Lenze_ST17H66 Tag or similar OpenHayStack Tags?

**Comment by Chapoly1305** [comment, marked as answer]
- I apologize, but I don’t have information to differentiate between the Lenze_ST17H66 and the actual AirTag. The Lenze_ST17H66 was sourced from an upstream repository, and I’ve removed them to avoid confusion. This repository primarily focuses on web API-related content.

- In the lost message, the Airtag transmits battery information but not the firmware information. You can refer to Apple’s specifications to gain a comprehensive understanding of the attributes transmitted in various scenarios.

- I would say yes, I believe there might be a few unused features, but I don’t observe them reflected in location reports. If you’re interested in data smuggling, I recall reading a paper about such a technique, but I can’t recall the details.

- Apple has implemented several measures to block OHS in different contexts. Firstly, Apple discontinued the Mail plugin, rendering the original OHS technique for receiving information ineffective on new platforms. Secondly, on December 2024, Apple released a patch to mitigate a vulnerability that prevented iPhone products from receiving and processing OHS like custom tags. This patch addresses an issue where the OHS Linux script used an incorrect address type for advertising, which is affected by the patch. Consequently, a portion of OHS tags may be impacted.
